grails-app/conf/Config.groovy

```diff
@@ -240,10 +226,26 @@ baselayers = [
 ]
 
 portal {
-    siteHeader = "[not configured]"
+    siteHeader = "Open Access to Ocean Data"
     motdUrl = "https://static.emii.org.au/motd"
-
-    logo = "images/favicon.ico"
+    logo = "https://static.emii.org.au/images/logo/AODN_logo_fullText.png"
+    secondaryLogo = "https://static.emii.org.au/images/logo/IMOS_logo-stacked-reversed-Converted.png"
+    icon = "https://static.emii.org.au/images/aodn/favicon.ico"
+
+    imos = "https://imos.aodn.org.au"
```

Comment:
This is config for configs sake.
There is only a few items that should be added to config
1) site wide email address
1) condition_of_use